src/Reveal.Sdk.Dom.Tests/Visualizations/OHLCVisualizationFixture.cs

```diff
@@ -0,0 +1,275 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Reveal.Sdk.Dom.Core.Serialization;
+using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Filters;
+using Reveal.Sdk.Dom.Visualizations;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations;
+
+public class OHLCVisualizationFixture
+{
+    [Fact]
+    public void Constructor_InitializesDefaultValues_WhenInstanceIsCreated()
+    {
+        // Act
+        var ohlcVisualization = new OHLCVisualization();
+
+        // Assert
+        Assert.NotNull(ohlcVisualization);
+        Assert.Equal(ChartType.OHLC, ohlcVisualization.ChartType);
+        Assert.Null(ohlcVisualization.Title);
+        Assert.Null(ohlcVisualization.DataDefinition);
+    }
+
+    [Fact]
+    public void Constructor_InitializesOHLCVisualizationWithDataSource_WhenDataSourceItemIsProvided()
```

Comment:
```suggestion
    public void Constructor_InitializesOHLCVisualizationWithDataSourceItem_WhenDataSourceItemIsProvided()
```

Reply:
Also need [Theory] to check for the case if `dataSourceItem` has `HasTabularData = false`